### PR TITLE
Unit Test for bulk InputEndpoint implementation.

### PR DESCRIPTION
**Pull request by anu3990**
So we can incorporate your pull request, please share the following:
* What issue are you addressing with this pull request? https://project.marklogic.com/jira/browse/JAVA-86
* Are you modifying the correct branch? develop-72
* Have you run unit tests? Yes
* Version of MarkLogic Java Client API 5.0.1
* Version of MarkLogic Server (see admin gui on port 8001)
* Java version 11.0.4
* OS and version
* What Changed: What happened before this change? What happens without this change?

### CODE REVIEW

File: marklogic-client-api/src/test/java/com/marklogic/client/test/dataservices/BulkInputCallerTest.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    static ObjectNode apiObj;
+    static String apiName = "bulkInputCallerImpl.api";
+    static String scriptPath;
+    static String apiPath;
+    static int workMax = 5;
+    static int startValue = 1;
+    static JSONDocumentManager docMgr;
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
```
Comment (ehennum) [inline marklogic-client-api/src/test/java/com/marklogic/client/test/dataservices/BulkInputCallerTest.java]: Would it be more robust to initialize expectedException to ExpectedException.none() within bulkInputEndpointTest()?

If someone added a test that expects an exception before bulkInputEndpointTest(), would that cause bulkInputEndpointTest() to fail?

Comment (anu3990) [reply marklogic-client-api/src/test/java/com/marklogic/client/test/dataservices/BulkInputCallerTest.java]: Hi Erik. I guess it should be okay. I tried adding a unit test to throw null pointer before bulkInputEndpointTest() and it ran successfully. Also, if we move the initialization of expectedException to bulkInputEndpointTest() we will need to initialize it again within bulkInputEndpointInterruptTest().
